src/connectors/admin.js

```diff
@@ -105,12 +105,15 @@ export default function (fetch, apiUrl) {
   }
 
   const accept = async function (formData) {
-    if (!formData || !formData.token || !formData.newPassword || !formData.newPasswordAgain) {
+    if (!formData || !formData.token || !formData.newPassword || !formData.newPasswordAgain || !formData.name) {
       throw new RouteError('Admin Password Is Required')
     }
     localStorage.setItem('accessToken', formData.token)
-    const res = await postAcceptedInvitaion({}, { newPassword: formData.newPassword, newPasswordAgain: formData.newPasswordAgain })
-    return res
+    const res = await postAcceptedInvitaion({}, { newPassword: formData.newPassword, newPasswordAgain: formData.newPasswordAgain, name: formData.name })
+    if (res.loginToken) {
+      localStorage.setItem('accessToken', res.loginToken)
```

Comment:
Hmm... why do you store the login token on the accessToken item in the local storage?